interfaces/automated-testing/scd/design/astm-strategic-coordination/nominal-planning-priority/sequence.plantuml

```diff
@@ -31,4 +44,17 @@ TD <- DSS: 200 Ok - returns Operational Intent Reference 2 with entityid2 manage
 TD -> USS2: GET /uss/v1/operational_intents/{entityid2}
 TD <- USS2: 200 Ok - returns Operational Intent 2
 
+...
+
+== TD attempts to transition flight in First-mover USS to Activated ==
+
+TD -> USS1: PUT /inject/flight\n{"state": "Activated"}
+TD <- USS1: 200 Ok - Flight update rejected\n{"result": "ConflictWithFlight"}
+
+TD -> DSS: POST /dss/v1/operational_intent_references/query\nwith area_of_interest encompassing Operational Intent 1
+TD <- DSS: 200 Ok or 404 Not found - Operational Intent Reference 1 is either unchanged or deleted
+
+TD -> USS1: GET /uss/v1/operational_intents/{entityid1}
+TD <- USS1: 200 Ok - Operational Intent 1 is unchanged (if not deleted in DSS)
```

Comment:
May I suggest to add the 404 status code here as well since it could be a valid outcome?

Reply:
Will do.